prow/cluster/pipeline-deployment.yaml

```diff
@@ -0,0 +1,40 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: prow-pipeline
+  namespace: default
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate 
+  selector:
+    matchLabels:
+      app: prow-pipeline
+  template:
+    metadata:
+      labels:
+        app: prow-pipeline
+    spec:
+      serviceAccount: prow-pipeline
+      containers:
+      - name: pipeline
+        image: gcr.io/k8s-prow/pipeline:v20190430-866dcd93f
+        args:
+        - --all-contexts
+        - --config=/etc/prow-config/config.yaml
+        - --kubeconfig=/etc/build-cluster/cluster
```

Comment:
Don't we want --build-cluster here not --kubeconfig?

https://github.com/kubernetes/test-infra/blob/863322699fe2bb1287dfe3b787ca4baec6da1cfe/prow/cluster/build_deployment.yaml#L25

Reply:
As I understand from this log message, `--build-controller` is deprecated in favour of `--kubeconfig`: https://github.com/kubernetes/test-infra/blob/4306ad7b3e0ca984dc608891f25a8f69bbe7e3bb/prow/cmd/build/main.go#L82

That't the reason why I use it.

Reply:
Yes but this isn't merely a flag rename. The file contents are different between the two formats.

Reply:
Got it. I changed the flag name to `build-cluster`.

Does the file provided to `kubeconfig` is a standard kubectl config?

Reply:
Yes, it should be a standard config